capnslog/README.md

```diff
@@ -0,0 +1,41 @@
+# CoreOS Log
+
+There are far too many logging packages out there, with varying degrees of licenses, far too many features (colorization, all sorts of log frameworks) or are just a pain to use (lack of `Fatalln()`?)
+
+## Design Principles
+
+* Logging is off by default, except critical failures, which are unrecoverable.
```

Comment:
On the flip side, having things log out from under you isn't great either; part of the reason to hijack stdlib log, even.